docs/scripts/load-async.js

```diff
@@ -0,0 +1,29 @@
+import enhanceCollection from "statinamic/lib/enhance-collection"
+
+const getFileFromContext = (context, filename) => {
+  const bundledResult = context.keys().map(context)
+  const result = enhanceCollection(bundledResult, {
+    filter: { __filename: filename },
+    limit: 1,
+  })
+
+  return result[0]
+}
+
+export default function loadAsync(filename, callback) {
+  if (__PROD__) {
+    const bundledResult = require("bundle!../content/" + filename)
```

Comment:
The problem is here. You will have to lazyload file.
You are not supposed to use dynamic require because webpack will bundle every thing in that folder in the bundle.
I think the require(bundle should be in the loader itself, but I am not sure, I am still using my brain to handle server side pre-rendering for dev (to be able to catch checksums issues).